### PR TITLE
Add decentralized-internet package

### PR DESCRIPTION
**Pull request by Mentors4EDU**
Hi, we have added our package to the Spack repository. Please let us know if you need anything else.

### CODE REVIEW

File: var/spack/repos/builtin/packages/decentralized-internet/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+##############################################################################
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class DecentralizedInternet(Package):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/decentralized-internet/package.py]:
```suggestion
class DecentralizedInternet(MakefilePackage):
```
This tells Spack that your package is just another makefile package, so you can remove the `install` method below, as it is the default for this base class.

File: var/spack/repos/builtin/packages/decentralized-internet/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+##############################################################################
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/decentralized-internet/package.py]: You can delete this line

**Comment by Mentors4EDU** [comment]
@adamjstewart Thanks so much, I committed your suggestion, ready to merge!

File: var/spack/repos/builtin/packages/decentralized-internet/package.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class DecentralizedInternet(MakefilePackage):
+    """A library for building decentralized and grid computing projects"""
+    homepage = "https://lonero.readthedocs.io"
+    url      = "https://github.com/Lonero-Team/Decentralized-Internet/releases/download/4.2.3/Decentralized.Internet.tar.gz"
+    maintainers = ['Lonero-Team', 'Mentors4edu']
+    version('4.2.3', sha256='2922b9128b411ece2f04d07942a453f1e772548aa27b3936c9f9bcfbc0737058')
+
+    def install(self, spec, prefix):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/decentralized-internet/package.py]: You can delete this function

File: var/spack/repos/builtin/packages/decentralized-internet/package.py (hunk 2 above)
Comment (Mentors4EDU) [reply var/spack/repos/builtin/packages/decentralized-internet/package.py]: Done

**Comment by Mentors4EDU** [comment]
@adamjstewart Anything else needed or this can be merged?

**Comment by adamjstewart** [comment]
Nope, just waiting for Travis CI to pass

**Comment by Mentors4EDU** [comment]
@adamjstewart Travis CI passed :)

**Pull request merged**